### DISCUSSION TITLE
Repositories aren't showing any language stats

### DISCUSSION THREAD

**Discussion by mohammadzainabbas** [github-linguist/linguist#5538] [Classification] [answered]
#### Describe the bug

Hi

I created two (private) repositories couple of days ago; and the language stats aren't appearing for both of them. From [here](https://github.com/github/linguist/blob/master/docs/overrides.md), it's understandable that low priority background jobs might take some time to finish; but it's been couple of days already.

Meanwhile I created another private repository, and the language stats for the newly created repo are showing (almost immediately).

#### Expected behaviour

Here is what I have tried so far:

- Pushed few commits to the same repo _(to restart jobs in case they were failed due to load on servers)_.
- Added [override rules](https://github.com/github/linguist/blob/master/docs/overrides.md) for linguist.

It's been couple of days since I pushed my commits, I expect to see language stats for my repo by now (especially after trying the above workarounds) 

#### Related discussion

Before creating this issue, I had gone through issues and tried to follow the details mentioned in them. (#4833 #4436 #3342 to name few), but so far nothing helped.

**Comment by lildude** [comment]
It's going to be impossible to explain what is happening without access to the repos. Are you able to reproduce the issue in a public repo and share the repo with us? Alternatively, are you able to add me as a collaborator and I can then have a look? This is almost certainly going to be due to the content or layout of your repos and not a bug.

**Reply by mohammadzainabbas** [reply]
@lildude I have sent you a collaboration invite to one of my private repo. You can have a look at it and let me know if I am missing something.

**Reply by lildude** [reply]
Thanks. I'm AFK right now but from a quick look, you have a lot of files in that repo. I suspect you might be hitting the same problem as the intellij-community repo discussed in https://github.com/github/linguist/discussions/5445 and the webkit repo discussed in https://github.com/github/linguist/discussions/5524 

I'll take a closer look tomorrow to confirm.

**Reply by mohammadzainabbas** [reply]
@lildude Yeah, number of files are bit large in both repos. Can I add override rules to run language stats of some of the sub-directories only ?? Does this guard check files after the override rules been applied or before ??

**Reply by lildude** [reply, marked as answer]
I've just checked and yes, this is definitely the same problem. Your repo has a _lot_ of files. Unfortunately, there is no override as the check is looking at the repo as a whole so the only option is to reduce the number of files in the repo.
